corehq/apps/app_manager/views/utils.py

```diff
@@ -125,3 +125,28 @@ def get_blank_form_xml(form_name):
         'xmlns': str(uuid.uuid4()).upper(),
         'name': form_name,
     })
+
+
+def overwrite_app(app, master_build, include_ucrs=False, report_map=None):
+    excluded_fields = set(Application._meta_fields).union(
+        ['date_created', 'build_profiles', 'copy_history', 'copy_of', 'name', 'comment', 'doc_type']
+    )
+    master_json = master_build.to_json()
+    for key, value in master_json.iteritems():
+        if key not in excluded_fields:
+            app[key] = value
+    app['version'] = master_json['version']
+    wrapped_app = wrap_app(app)
+    errors = False
+    for module in wrapped_app.modules:
+        if isinstance(module, ReportModule):
+            if include_ucrs and report_map is not None:
+                for config in module.report_configs:
+                    config.report_id = report_map[config.report_id]
+            else:
+                errors = True
```

Comment:
Just a style point here. There are two better options:
1. `return True` which short circuits the rest of the function and means you don't have to check `errors` later
2. Raise an exception. This is usually the best approach when a function wants to fail. It also makes it clear that the function has failed and what the error is.